src/Nox.Reference.GetData/CliCommands/ICliCommandExecutor.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Nox.Reference.GetData.CliCommands;
+
+public interface ICliCommandExecutor
```

Comment:
Consider each data extractor as cli command which can be run independently.

Reply:
We should probably have a subtle executable with multiple command support.  Use Spectre Console/Cli as we do in Nox.Cli. but let's just add this as an issue on github please then discuss and if we agree implement at end of critical services